src/java.base/share/classes/javax/crypto/ExemptionMechanism.java

```diff
@@ -139,7 +139,7 @@ public final String getName() {
      *
      * @see java.security.Provider
      */
-    public static final ExemptionMechanism getInstance(String algorithm)
+    public static ExemptionMechanism getInstance(String algorithm)
```

Comment:
Others might disagree with this comment, but I would encourage you not to make these changes throughout your PR.  (@jddarcy ?) 

Even though a static method is implicitly final and IJ is correct, the final keyword does prevent subclasses from inadvertently using the same method signature.

Reply:
Keeping the original here and elsewhere.